string row
int32 column
string override